src/wallet/wallet.ts

```diff
@@ -122,18 +136,130 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     // TODO should we have a debug mode?
   }
 
+  /**
+   * Remove sensitive data from memory
+   *
+   * NOTICE: This won't remove data from memory immediately, we have to wait until javascript
+   * garbage collect it. JavaScript currently does not provide a standard way to trigger
+   * garbage collection
+   **/
+  clearSensitiveData() {
+    this.seed = null;
+    this.authPrivKey = null;
+  }
+
+  /**
+   * Get auth xpubkey in the 280' purpose path from seed
```

Comment:
I'd say "Get auth xpubkey from seed".

Reply:
I agree, the docstring was confusing

Done in 9c7185a0b401fea8803a5dff09c70257f5f09963

Reply:
Great! Thanks!